.github/ISSUE_TEMPLATE/7_cherry_pick.yml

```diff
@@ -38,7 +38,7 @@ body:
       label: Changelog Description
       description: >-
         Explain this CP in less than 80 characters that is accessible to most Flutter developers
-        See https://github.com/flutter/flutter/wiki/Hotfix-Documentation-Best-Practices for examples
+        See https://github.com/flutter/flutter/blob/main/docs/releases/Hotfix-Documentation-Best-Practices.md for examples
```

Comment:
Do we really want to link to `main` everywhere? Don't edits still have to be made to `master`, making it harder to edit if we link to `main`?

Reply:
Yeah, this really annoyed me. I feel like if we put `main`, we're more likely to get PRs opened against `main`.
However, our analyzer script (only in flutter/flutter) enforces using `main` and does not allow `master`. 

When I first put up this PR is was all master, and then I changed it to main to make CI happy. We don't do it that other repos. It feels like a footgun for contributors. 🤷

Reply:
> However, our analyzer script (only in flutter/flutter) enforces using main and does not allow master.

Can we change the script instead?

Reply:
> > However, our analyzer script (only in flutter/flutter) enforces using main and does not allow master.
> 
> Can we change the script instead?

Oh can we? It looks like it's fairly recent, added in https://github.com/flutter/flutter/pull/146882 as part of https://github.com/flutter/flutter/issues/121564. But if that issue is not actively being worked on, maybe we can revert this analyzer bit.

There is already quite a list of exempt repos.. because they don't have a main branch.

https://github.com/flutter/flutter/blob/a92318dd9822edbf0e2d4bcffb923fcd54118c8a/dev/bots/analyze.dart#L1303-L1320

Reply:
Lemme take a look

Reply:
Added 'flutter/flutter' with a TODO and link to the tracking issue, which is currently unassigned

Reply:
Circling back on this, I won't change all of the instance of `main` in this PR, since that will put us in a mixed state with all the other `main` links we previously enforced. I'll send another PR that switches all instances back to `master` if that is what we'd like to do.